{
  "session_id": "crawl_1749619435_ce2f3c66",
  "institution_name": "Bank of America California",
  "institution_type": "bank",
  "timestamp": 1749619435.2747972,
  "urls_requested": [
    "https://web.marioncc.org/wcdirectory/directory.aspx?listingid=4074&profileid=3J1S1B3P5A8R14&action=uweb&url=http%3A%2F%2Fwww.countrywide.com",
    "https://accounts.nfhs.org/users/single_logout?redirect_to=https://about.bankofamerica.com/en/making-an-impact/museums-on-us-partners",
    "https://cgpct.org/BankOfAmericaCharitable",
    "https://www.fdic.gov/bank-failures/failed-bank-list",
    "https://en.wikipedia.org/wiki/Bank_of_America"
  ],
  "total_crawl_time": 41.51866793632507,
  "success": true,
  "urls_successful": 2,
  "urls_failed": 1,
  "cache_hits": 0,
  "api_calls": 0,
  "total_content_size_bytes": 944770,
  "total_word_count": 26711,
  "average_quality_score": 0.0,
  "url_benchmarks": [
    {
      "url_benchmark_id": "url_1749619435_adfa906f",
      "session_id": "crawl_1749619435_ce2f3c66",
      "url": "https://cgpct.org/BankOfAmericaCharitable",
      "timestamp": 1749619435.9520795,
      "crawl_time": 30.35689663887024,
      "success": false,
      "content_size_bytes": 0,
      "word_count": 0,
      "quality_score": 0.0,
      "data_types_extracted": {},
      "content_format_sizes": {},
      "media_counts": {},
      "institution_data_quality": {},
      "error": "Crawl failed: Unexpected error in _crawl_web at line 744 in _crawl_web (nlp\\Lib\\site-packages\\crawl4ai\\async_crawler_strategy.py):\nError: Failed on navigating ACS-GOTO:\nPage.goto: Timeout 30000ms exceeded.\nCall log:\n  - navigating to \"https://cgpct.org/BankOfAmericaCharitable\", waiting until \"domcontentloaded\"\n\n\nCode context:\n 739                       response = await page.goto(\n 740                           url, wait_until=config.wait_until, timeout=config.page_timeout\n 741                       )\n 742                       redirected_url = page.url\n 743                   except Error as e:\n 744 →                     raise RuntimeError(f\"Failed on navigating ACS-GOTO:\\n{str(e)}\")\n 745   \n 746                   await self.execute_hook(\n 747                       \"after_goto\", page, context=context, url=url, response=response, config=config\n 748                   )\n 749   ",
      "status_code": null,
      "crawl4ai_performance": {}
    },
    {
      "url_benchmark_id": "url_1749619466_25c83d03",
      "session_id": "crawl_1749619435_ce2f3c66",
      "url": "https://www.fdic.gov/bank-failures/failed-bank-list",
      "timestamp": 1749619466.3089762,
      "crawl_time": 5.977358818054199,
      "success": true,
      "content_size_bytes": 103851,
      "word_count": 2297,
      "quality_score": 0.0,
      "data_types_extracted": {},
      "content_format_sizes": {},
      "media_counts": {},
      "institution_data_quality": {},
      "error": null,
      "status_code": null,
      "crawl4ai_performance": {}
    },
    {
      "url_benchmark_id": "url_1749619472_7e53f076",
      "session_id": "crawl_1749619435_ce2f3c66",
      "url": "https://en.wikipedia.org/wiki/Bank_of_America",
      "timestamp": 1749619472.2967036,
      "crawl_time": 4.287724256515503,
      "success": true,
      "content_size_bytes": 840919,
      "word_count": 24414,
      "quality_score": 0.0,
      "data_types_extracted": {},
      "content_format_sizes": {},
      "media_counts": {},
      "institution_data_quality": {},
      "error": null,
      "status_code": null,
      "crawl4ai_performance": {}
    }
  ],
  "domains_crawled": {
    "cgpct.org": 1,
    "www.fdic.gov": 1,
    "en.wikipedia.org": 1
  },
  "errors": [
    "https://cgpct.org/BankOfAmericaCharitable: Crawl failed: Unexpected error in _crawl_web at line 744 in _crawl_web (nlp\\Lib\\site-packages\\crawl4ai\\async_crawler_strategy.py):\nError: Failed on navigating ACS-GOTO:\nPage.goto: Timeout 30000ms exceeded.\nCall log:\n  - navigating to \"https://cgpct.org/BankOfAmericaCharitable\", waiting until \"domcontentloaded\"\n\n\nCode context:\n 739                       response = await page.goto(\n 740                           url, wait_until=config.wait_until, timeout=config.page_timeout\n 741                       )\n 742                       redirected_url = page.url\n 743                   except Error as e:\n 744 →                     raise RuntimeError(f\"Failed on navigating ACS-GOTO:\\n{str(e)}\")\n 745   \n 746                   await self.execute_hook(\n 747                       \"after_goto\", page, context=context, url=url, response=response, config=config\n 748                   )\n 749   "
  ]
}